image/remote.go

```diff
@@ -258,6 +261,8 @@ func (r *remote) Save() (string, error) {
 		return "", err
 	}
 
+	log.SetOutput(ioutil.Discard)
```

Comment:
Mixed feelings about this.

Upside:
- Probably no other way to do this without an upstream PR
- `log.SetOutput` is re-entrant, so a global data race isn't possible

Downside:
- This temporarily disables logging for any functions running in parallel that use the global logger (and the global logger is designed to be used in parallel). This could introduce a very surprising bug: some unrelated logs could occasionally disappear.

Would it make more sense to move this logic into a `main` package? If naively moving it up the stack would disable too many logs, could we fix this by inverting control?